ops/terraform/services/base/values/prod.yaml

```diff
@@ -570,6 +570,14 @@ bfd:
+        o11y:
+          alarms:
+            lambda_errors:
+              breach_topic: <<CIPHER>>AgV4EMDbvi4QHKhF2NB8G8K9kpXDwOeBwWxfGYI970egWxEAdAACAAtBcHBsaWNhdGlvbgAGY2lwaGVyABVhd3MtY3J5cHRvLXB1YmxpYy1rZXkAREFoT1RHMWpmTXFsYW5udENEb08rcFVtU2crVWp1UEhMTHY2SlJ2c1BYRFZEQ2tNMjMzMXB5Z3plOWZpQm5uSktUZz09AAEAB2F3cy1rbXMAS2Fybjphd3M6a21zOnVzLWVhc3QtMTo1NzczNzM4MzE3MTE6a2V5L21yay1iMjNlYjZmZWVlYmM0MzQ2YTk5YTZhNTliYzU2MWRmMwC4AQIBAHipsbBUtIKaXI95_AIHiGnjDkxfPDRzgqb6WVwMZI7-kwH9g_UZkuezbQyNh3Ia-RbuAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMuu9L8yHsEUmtIx8mAgEQgDvdIX5KlTpkAINHtUSle2VUi6swbuk_tUHgrmBxSrIkYjNQzK2pIy80p4yTtGmq0ucM2Xil3CdbwLFkeQIAABAASQVjd0NrvurOv0bA-DpBAw555bhbluMSwUJ2PEVciG-FQbkU_Zv8sFI_s7oJE3Rf_____wAAAAEAAAAAAAAAAAAAAAEAAAAczFGqF_X6g6iIpuw-UxY84hIgMiw36kac8YwDPWo0sgbwH19l-9qePN412DMAZzBlAjAjkWYXWTduHyda9D_gd2N1J5DIhX0N2ci1e2tRyPyfKyDrfJBs8mBBSTDYGAPqhiQCMQD6-x1xmRyEh3vEUG0lIvOSe2HCsH2pQBmpgMF8Tv-emHPz2m_lbME3pwA3EGXCq-M=<</CIPHER>>
```

Comment:
@malessi non-blocking, just for my understanding: is this supposed to be the encrypted arn or encrypted name of the topic?

Reply:
@meliGuzman It's merely conjecture, but I would guess that the solution originally used fully-qualified ARNs for the SNS Topics that **do** include the semi-sensitive AWS Account Number rather than _just_ the topic name alone that we see today. I gather that you don't see anything worth treating as sensitive, and I would agree with that.

While this doesn't necessarily need to block acceptance of the change, maybe @malessi could offer an explanation?

Reply:
Your conjecture would be mostly correct. I decided to switch to names instead of ARNs to make the configuration a little friendlier, but didn't opt to update the _sensitivity_ of these values (mostly out of convenience). 

I can make these nonsensitive, if either of you feel that it'd make understanding the configuration a little easier.

Reply:
@malessi  I don't see it as a blocker but if it doesn't take much effort then can you change them to nonsensitive? Just for the sake of following some standard

Reply:
I've gone ahead and moved these parameters to nonsenstive instead of sensitive, as I agree that they're really not sensitive and should be configured as such.